site/yarn.lock

```diff
@@ -1771,6 +1771,13 @@
   dependencies:
     "@types/node" "*"
 
+"@types/cssnano@^4.0.1":
+  version "4.0.1"
+  resolved "https://registry.yarnpkg.com/@types/cssnano/-/cssnano-4.0.1.tgz#67fa912753d80973a016e7684a47fedf338aacff"
+  integrity sha512-hGOroxRTBkYl5gSBRJOffhV4+io+Y2bFX1VP7LgKEVHJt/LPPJaWUIuDAz74Vlp7l7hCDZfaDi7iPxwNwuVA4Q==
+  dependencies:
+    postcss "5 - 7"
```

Comment:
Hm, interesting, I think we really need types here, as minimum for public functions

Reply:
Does it mean we need to add TypeScript with TSDoc comments?
I've also wondered why that package appeared. Seems like `css-minimizer-webpack-plugin` is pulling it in.
=> Found "@types/cssnano@4.0.1"
info Reasons this module exists
   - "@docusaurus#core#css-minimizer-webpack-plugin" depends on it
   - Hoisted from "@docusaurus#core#css-minimizer-webpack-plugin#@types#cssnano"

Reply:
> Does it mean we need to add TypeScript with TSDoc comments?

Yes, it will be enough